Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SystemType.cs

```diff
@@ -24,12 +27,9 @@ public static string GetReference(Type type)
                 return string.Empty;
             }
 
-#if WINDOWS_UWP && !ENABLE_IL2CPP
-            return $"{type.FullName}, {type.GetAssembly().GetName().Name}";
-#else
-            return $"{type.FullName}, {type.Assembly.GetName().Name}";
-#endif
-
+            string[] qualifiedNameComponents = type.AssemblyQualifiedName.Split(new char[] { ',' });
+            Debug.Assert(qualifiedNameComponents.Length >= 2);
```

Comment:
What's with this?  This change breaks my serialization for the this.

Reply:
The reason was to eliminate a platform specific block.

Reply:
How did the serialization break? This should be returning the same string as the other version.

Reply:
because each platform is referencing the assembly a bit differently is seems.

Reply:
it might be because there's some hidden characters added/removed in this.

Reply:
I just need to update the profiles with the updated serialization and it should be fine.